browser/ui/brave_actions/brave_action_view_controller.cc

```diff
@@ -87,19 +87,44 @@ ui::MenuModel* BraveActionViewController::GetContextMenu(
   return nullptr;
 }
 
-bool BraveActionViewController::ExecuteActionUI(
-    std::string relative_path) {
-  return TriggerPopupWithUrl(PopupShowAction::SHOW_POPUP,
-      extension()->GetResourceURL(relative_path),
-      true);
+void BraveActionViewController::ExecuteActionUI(
+    const std::string& relative_path) {
+  TriggerPopupWithUrl(PopupShowAction::SHOW_POPUP,
+                      extension()->GetResourceURL(relative_path),
+                      /*grant_tab_permissions=*/true);
 }
 
 ExtensionActionViewController*
 BraveActionViewController::GetPreferredPopupViewController() {
   return this;
 }
 
-bool BraveActionViewController::TriggerPopupWithUrl(
+void BraveActionViewController::TriggerPopup(PopupShowAction show_action,
+                                             bool grant_tab_permissions) {
+  content::WebContents* const web_contents =
```

Comment:
this can be null 
```// Returns the currently active WebContents, or NULL if there is none.
  content::WebContents* GetActiveWebContents() const;```

Reply:
Fixed in https://github.com/brave/brave-core/commit/178529b2929c35960d03be82e8e32e1124dfa00e